ivy_tests/test_ivy/test_frontends/test_torch/test_random_sampling.py

```diff
@@ -458,3 +458,36 @@ def call():
     for u, v in zip(ret_np, ret_from_np):
         assert u.dtype == v.dtype
         assert u.shape == v.shape
+
+
+# set_rng_state
+@handle_frontend_test(
+    fn_tree="set_rng_state",
+    state=st.shared(
```

Comment:
using `st.shared` for the `state` parameter means that the generated random state array will be shared across multiple test cases. This can be useful at times if one wants to ensure that the same state is used consistently for a group of related test cases.  But if not needed, i can perhaps remove and update the code